test/lib/client.go

```diff
@@ -61,8 +60,8 @@ type Client struct {
 
 	podsCreated []string
 
-	tracingEnv corev1.EnvVar
-	loggingEnv *corev1.EnvVar
+	TracingCfg string
+	LoggingCfg string
```

Comment:
Yes. It was for that reason. I somehow find it cleaner if the client holds the config instead of an EnvVar. The client already holds rest.Config for example. It can then create the EnvVar (which is only done in a single place anyway) or it can pass the config to Wathola. But I don't have a strong opinion about this.

Reply:
I'm not forcing this to rollback.